exporter.go

```diff
@@ -247,6 +252,15 @@ For NGINX, the stub_status page must be available through the URI. For NGINX Plu
 	sslVerify = flag.Bool("nginx.ssl-verify",
 		defaultSslVerify,
 		"Perform SSL certificate verification. The default value can be overwritten by SSL_VERIFY environment variable.")
+	sslCaCert = flag.String("nginx.ssl-ca-cert",
+		defaultSslCaCert,
+		"The CA certificate to use to validate the servers SSL certificate. The default value can be overwritten by SSL_CA_CERT environment variable.")
+	sslClientCert = flag.String("nginx.ssl-client-cert",
+		defaultSslClientCert,
+		"The client certificate to use when connecting to the server. The default value can be overwritten by SSL_CLIENT_CERT environment variable.")
+	sslClientKey = flag.String("nginx.ssl-client-key",
+		defaultSslClientKey,
+		"The client certificate key to use when connecting to the server. The default value can be overwritten by SSL_CLIENT_KEY environment variable.")
```

Comment:
Maybe it's better to say something like "Path to the client certificate file.."? To make it clearer that you need to pass the file path.